src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -247,13 +247,13 @@ $theme-colors: (
 
 // Typography
 $base-font-size: 16px;
-$small-font-size: calculateRem(14px) !default;
-$h1-font-size: calculateRem(28px) !default;
-$h2-font-size: calculateRem(22px) !default;
-$h3-font-size: calculateRem(18px) !default;
-$h4-font-size: calculateRem(16px) !default;
-$h5-font-size: calculateRem(14px) !default;
-$h6-font-size: calculateRem(12px) !default;
+$small-font-size: 14px !default;
```

Comment:
Just question: we do not want `calculateRem` here? Why?

Reply:
Boostrap is multiplying those values and it triggers an error.

Reply:
What about fixing out `calculareRem`? It does not convert values properly, will fixing it fix this bootstrap problem?

Fixed`calculateRem`:
```scss
@function calculateRem($size) {
    $remSize: $size / $base-font-size;

    @return $remSize + 0rem;
}
```

Reply:
discussed in private